skins/oasis/modules/templates/Body_Index.php

```diff
@@ -137,7 +137,7 @@
 					<?= $bodytext ?>
 
 				</div>
-				<?php if ( !(new ARecoveryModule)->isSourcePointRecoveryDisabled() ) { ?>
+				<?php if ( (new ARecoveryModule)->isSourcePointRecoveryEnabled() ) { ?>
```

Comment:
What do you think of making it consistent and using "PHP template conditions" instead of brackets everywhere in the file? :japanese_ogre: